blackbox/docs/protocols/crate-standard-sql.txt

```diff
@@ -0,0 +1,153 @@
+.. _crate_standard_sql:

```

Comment:
i think this file should go somewhere here: https://crate.io/docs/reference/sql/index.html